### PR TITLE
Feature/tvist1 257 adding case parties merge

### PR DESCRIPTION
**Pull request by jekuaitk**
* Added PartyController
* Added CasePartyRelation
* Added CasePartyRelationListener
* Added PartyListener
* Added PartyHelper
* Added AddPartyFromIndexType
* Added PartyFormType
* Added twig templates for party `add`, `add_from_party_index`, and `edit`

Edit: 
* Added `CaseHelper` to provide template path and complainants/counterparties

### CODE REVIEW

File: templates/party/add.html.twig [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
+{% block title %}{% trans %}Add Party{% endtrans %}{% endblock %}
+
```
Comment (lats1) [inline templates/party/add.html.twig:4]: Does this need a translation domain?

File: templates/party/add_party_from_index.html.twig [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
```
Comment (lats1) [inline templates/party/add_party_from_index.html.twig:2]: Does this need a translation domain?

File: templates/party/edit.html.twig [hunk 3]
```diff
@@ -0,0 +1,54 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
```
Comment (lats1) [inline templates/party/edit.html.twig:2]: Does this need a translation domain?

**Review by lats1 — CHANGES_REQUESTED** [review]

File: templates/party/add.html.twig (hunk 1 above)
Comment (jekuaitk) [reply templates/party/add.html.twig:4]: Added translation domain and made translations

File: templates/party/add_party_from_index.html.twig (hunk 2 above)
Comment (jekuaitk) [reply templates/party/add_party_from_index.html.twig:2]: Added translation domain and made translations

File: templates/party/edit.html.twig (hunk 3 above)
Comment (jekuaitk) [reply templates/party/edit.html.twig:2]: Added translation domain and made translations

File: src/Controller/CaseController.php [hunk 4]
```diff
@@ -40,10 +41,30 @@ public function summary(CaseEntity $case): Response
...
+
+        foreach ($landlordRelations as $relation) {
+            array_push($landlords, $relation->getParty());
+        }
+
         return $this->render('case/show.html.twig', [
             'case' => $case,
+            'tenants' => $tenants,
+            'landlords' => $landlords,
         ]);
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/Controller/CaseController.php:68]: Added `CaseHelper` service with `getRelevantTemplateAndPartiesByCase()` function which returns an array containing a template path, an array of complainants and an array of counterparties.

**Review by lats1 — APPROVED** [review]

**Pull request merged**